### ISSUE TITLE
Entire site A11Y: Right-side menu/list of links

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#158]
**Describe the bug**
On all pages that use that left-side navigation menu/list of links, those links are not easy to find for someone using a screen reader or someone on a mobile divide. For these people, those links appear at the very bottom of the web page, which makes them hard to find. This is especially a problem for the "Browse Our Collection" page, as someone would have to go through all the books on the page to find where there they can select specific subject categories. 

And looking at different web pages on my phone, many of them I cannot find those list of links at all. 

**To Reproduce**
1. Go to 'https://helga.bccampus.ca/wp/opentextbooks/2-0-find-open-textbooks/browse-our-collection/
2. Activate screen reader.
3. Try to figure out how to go through the categories that appear at the top of the screen.
4. Fail miserably.

**Expected behavior**
I expect that those links should be easy to find for someone using a screen reader or someone on a mobile device.

**Screenshots**
This shows the location of the collection categories (at the bottom of the search results) on my Google Pixel phone in Chrome.
![mobileview](https://user-images.githubusercontent.com/23706762/54936719-4c39ad80-4ee0-11e9-8ed3-7b341c854049.png)

**Comment by alex-418** [comment]
Change the menu on mobile to a drop-down style, move to a location (before the content) that works for all pages.

**Comment by alex-418** [comment]
@josieg @ehbeattie completed this via #161. You should now see a new menu after the breadcrumbs on mobile.... with the heading "Browse Subjects". 

Can you please test with a screen reader to see if the desired results were achieved? Please let me know of any other changes you'd like to this. Thank you. 

![mobile_menu](https://user-images.githubusercontent.com/17072191/54954173-473c2480-4f07-11e9-9811-721c20e0cc1c.png)
![expanded](https://user-images.githubusercontent.com/17072191/54954181-499e7e80-4f07-11e9-9b67-7ff93a7e84c2.png)

**Comment by josieg** [comment]
Hi Alex, on my phone I am now seeing the "Subjects" drop-down menu on all pages, not just the "Browse Open Textbooks" page. For example, when I click the "Use Open Textbooks" page, I see a "Browse subjects" drop-down menu rather than the "Use open textbooks" drop-down menu.

**Comment by alex-418** [comment]
Thanks @josieg I will limit this new subjects menu to the following pages: 
- Find Open Textbooks
- Individual textbook page 

I think that's the only places it should show up?

**Comment by josieg** [comment]
I think it should be:
- Browse Our Collection: https://helga.bccampus.ca/wp/opentextbooks/2-0-find-open-textbooks/browse-our-collection/
- Individual textbook page

**Comment by josieg** [comment]
Still no success finding the right-side menu with a screen reader

**Comment by alex-418** [comment]
@josieg can you BlueJeans or Skype? I'd like to see what the screen reader sees.

**Comment by josieg** [comment]
Yep. The screen reader doesn't really show anything visually, but I can demonstrate what it says

**Comment by alex-418** [comment]
We tested this with @josieg and while it works great for mobile, the desktop version requires the menu to be positioned BEFORE the content in the markup, this is so that the tabbing order is useful for when navigating with a screen reader.

This means moving the menu to the left on the "Browser our collection" page, and on individual book pages. If that's the direction we'd like to go, I can update the template. Here's a screenshot of what it would look like: 

![lefty](https://user-images.githubusercontent.com/17072191/54960960-1fa38700-4f1c-11e9-8619-18d39539b2f4.png)

**Comment by ehbeattie** [comment]
Lovely. Good to go!
